### ISSUE TITLE
dchudz won

### ISSUE THREAD

**Issue by jeffkaufman** [jeffkaufman/nomic#83]
We flipped a coin over who would give points to whom.  He won, so I approved #82 which give him 100k points and took 200k points from me.

<img width="461" alt="screen shot 2019-01-29 at 9 09 48 am" src="https://user-images.githubusercontent.com/262566/51913925-aa955480-23a5-11e9-8798-8b4ac1710349.png">

**Comment by jeffkaufman** [comment]
congratulations @dchudz !

**Comment by tnelling** [comment]
You're too honest for your own good, Jeff.

**Comment by tnelling** [comment]
If we want to prevent such self-sacrificing shenanigans in the future, we should floor transfer points at 0. Probably should have anyway.

**Comment by jeffkaufman** [comment]
bugs this shows:

* no one should be allowed to go below 0 points
* prs that add players shouldn't count as points transfer prs

**Comment by jeffkaufman** [comment]
after david and I had already agreed to do this we realized that someone could win on their own by using points transfer prs to add enough sock puppets to become a supermajority and then approve a PR that made them win explicitly

**Comment by tnelling** [comment]
I don't think disallowing going below 0 generally is necessary, but no transfer PRs should be allowed to go below 0, or remove points from a player with negative points.

**Comment by tnelling** [comment]
I didn't say Jeff (or you) was too trusting, I said he was too honest. Once the coin flip happened, there was no benefit to him in following through beyond reputation. I certainly agree the bug could have been exploited in other ways.

**Comment by dchudz** [comment]
Another way to win unilaterally would be by giving negative points to a sock puppet.

Once I realized that, I considered trying to win on my own, but wasn't sure what my agreement with Jeff was. After talking we decided to flip the coin.

Also I was a little confused about the process of winning unilaterally. In particular, I thought a non-player like me could do it, but that's not true b/c:

- the person losing points needs to approve the PR
- approvals are only queried for players

So once I was a player I could have added a socket puppet unilaterally (by transferring a point to them), and the sock puppet could have let me win. But I'd have needed to be a player first.

**Comment by dchudz** [comment]
> Once the coin flip happened, there was no benefit to him in following through beyond reputation.

Yeah. I agree it that for some purposes it'd be nice to make inter-game reputation not matter, but I don't know how to do that except completely anonymity. (Some people play diplomacy that way, right?)  So I mostly treat it as part of the game.

I realize people have very different views/approaches on that.

**Comment by jeffkaufman** [comment]
Wrote a post summarizing the win: https://www.jefftk.com/p/nomic-report-iii-conclusion

**Comment by pavellishin** [comment]
> *it seemed too likely that people would realize the account didn't have a proper hsitory and reject the join PR.*

I, for one, would almost certainly not have bothered checking :)

**Comment by pavellishin** [comment]
> *diplomacy*

Or maybe we could pivot from nomic to this :)

(Although I loathe, fucking *loathe* the diplomacy map, which concentrates all of the interesting stuff in a very hard to read center section. Maybe that's part of the point, but for the most part it's just annoying and fiddly, and really turned me off the first and only time I played.)

**Comment by tnelling** [comment]
Your opinion on Diplomacy is unenlightened :P

**Comment by jeffkaufman** [comment]
I've been wanting to GM another game of https://www.jefftk.com/p/dip-space-nine for a while...

**Comment by pavellishin** [comment]
@tnelling I'm willing to listen to a sales pitch on why my opinion is dumb, it's the only way to get rid of dumb opinions

**Comment by dchudz** [comment]
I'd probably be interested in Diplomacy, especially the one @jeffkaufman wants to GM.

**Comment by pavellishin** [comment]
Thread created: https://github.com/jeffkaufman/nomic/issues/84
